crates/agent/src/directives/mod.rs

```diff
@@ -40,11 +40,18 @@ pub enum Directive {
     Grant(grant::Directive),
 }
 
-pub struct DirectiveHandler {}
+#[derive(Default)]
+pub struct DirectiveHandler {
+    tenant_template: models::Catalog,
+    accounts_user_email: String,
```

Comment:
I see this passed around a bunch... this is `accounts@estuary.dev` right? Should this just be a global value anyone can reference? Maybe based on env vars or something? It doesn't seem like it'll ever change during the lifecycle of any of these crates

Reply:
Yep, `accounts@estuary.dev` or `accounts@company-who-self-hosts.com`, etc.